bokeh/core/property/bases.py

```diff
@@ -61,9 +61,6 @@ def __init__(self, default=None, help=None, serialized=True, readonly=False):
         self.alternatives = []
         self.assertions = []
 
-        # "fail early" when a default is invalid
-        self.validate(self._raw_default())
```

Comment:
This is unfortunate, but it's the more reliable way to get around Python's recursive imports. On the bright side, removing this makes `from bokeh.models import *` 20% faster. To make sure all defaults are of the correct type, a test can be added.

Reply:
I do agree that this kind of testing should happen ex-runtime, so that user's don't have to pay for it.